src/BloomExe/XmlHtmlConverter.cs

```diff
@@ -96,6 +96,10 @@ public static XmlDocument GetXmlDomFromHtml(string content, bool includeXmlDecla
 						// The regex here is mainly for the \s as a convenient way to remove whatever whitespace TIDY
 						// has inserted. It's a fringe benefit that we can use the[bi] to deal with both elements in one replace.
 						newContents = Regex.Replace(newContents, @"REMOVEWHITESPACE\s*\<([biu])\>", "<$1>");
+
+						// remove blank lines at the end of style blocks
+						newContents = Regex.Replace(newContents, @"\s+\<(\/style)\>", "<$1>");
```

Comment:
This is puzzling. It would be natural to leave out the capture parens and just make the output </style>. Or to capture the whole thing you want to keep by moving the parens and the angle brackets in the output, to emphasize that all you're doing is removing the \s+. But why use $1 for only part of what you aren't changing?